web/src/engine/websites/decorators/CoreView.ts

```diff
@@ -0,0 +1,267 @@
+import { FetchRequest, FetchCSS } from '../../FetchProvider';
+import { type MangaScraper, type MangaPlugin, Manga, Chapter, Page } from '../../providers/MangaPlugin';
+import type { Priority } from '../../taskpool/TaskPool';
+import * as Common from './Common';
+
+export const mangaPaths = ['/series', '/series/oneshot', '/series/finished'];
+export const queryMangaTitleFromURI = '.series-header-title';
+
+// The query to select *all* required data for *one* manga
+export const queryMangas = [
+    'article.series-list-wrapper ul.series-list > li.series-list-item > a',
+    'article.serial-series-contents ul.serial-series-list > li.serial-series-item > a',
+    'div.series-items ul.daily-series > li.daily-series-item > a',
+    'ul.webry-series-list > li.webry-series-item > a',
+    'section.series ul.series-series-list > li.series-series-item',
+    'div.serial-contents ul.series-table-list > li.subpage-table-list-item > a',
+    'div.series ul.series-section-list li.series-section-item a.series-section-item-link',
+    'div.yomikiri-container ul.yomikiri-items > li.yomikiri-item-box > a.yomikiri-link',
+    'div.serial-contents section div.series-item h4 > a',
+    'section.series-section ul.series-items > li.series-item > a',
+].join(',');
+
+const queryMangaURI: string = undefined;
+// The query to retrieve the single manga title from inside of this.queryManga
+export const queryMangaTitle = [
+    '.series-list-title',
+    '.series-title',
+    'h4.daily-series-title',
+    '.item-series-title'
+].join(',');
+
+const queryChaptersAtomFeed = 'head link[type*="atom+xml"]';
+export const queryChapters = 'feed entry';
+export const queryEpisodeJSON = '#episode-json';
+
+type ChapterJSON = {
+    readableProduct: {
+        isPublic: boolean,
+        hasPurchased: boolean,
+        pageStructure: {
+            choJuGiga: string,
+            pages : {
+                type: string,
+                src: string
+            }[]
+        }
+    }
+}
+function MangaExtractor(element: HTMLElement, queryMgURI: string, queryMgtitle: string) {
```

Comment:
The term `Mg` seems to be an abbreviation, suggestion to use the full term or remove it.

Reply:
See other occurrences as well